submitted_models/marble_spot_sensor_config_1/specifications.md

```diff
@@ -0,0 +1,90 @@
+# Boston Dynamics Spot - CTU-CRAS-Norlab sensor config
+This specifications.md file is a description and proof of virtual model validation for the
+Boston Dynamics Spot robot configured by team CTU-CRAS-Norlab. This robot may be launched using an
+ign launch command with the variable name `marble_spot_sensor_config_1`.
+
+For local testing, you can use the following command:
+
+    LC_ALL=C ign launch -v 4 ~/subt_ws/src/subt/submitted_models/marble_spot_sensor_config_1/launch/example.ign robotName:=X1 ros:=true champ:=true
+
+This specifications.md file describes mainly the additional sensory payload of our Spot. Refer to
+[bosdyn_spot/specifications.md](../bosdyn_spot/specifications.md) for information about the robot base,
+how to control it and how to modify parameters of the robot.
+
+## Description
+This sensor config carries 3 wide-angle RGB cameras with global shutter, an IMU and a 3D lidar.
+
+## Usage Instructions
+
+See [bosdyn_spot/specifications.md](../bosdyn_spot/specifications.md). There is no diversion from
+the principles described there, just substitute `bosdyn_spot` with `marble_spot_sensor_config_1`.
+
+## Usage Rights
+No additional restrictions have to be taken into account for this model.
+
+## Cost and Scale
+* Base vehicle: $75,000
+* Flir Cameras + Lenses: $2,500
+* IMU: $1,500
+* Computers: $1,500
+* Lidar: $6,000
+* Total price: $88,500
+
+vehicle weight: TODO
+
+## Sensors
+This sensor configuration of Spot includes the following sensors in addition to the base sensors of the platform
+described in [bosdyn_spot/specifications.md](../bosdyn_spot/specifications.md).
+The specifications for these instruments are provided below in the [Validation Links](#validation_links) section.
+
+The following specific sensors are declared payloads of this vehicle:
+
+* Ouster OS1-Gen1 3D lidar modeled by `gpu_lidar` sensor. It runs configured to produce 1024x64 scans at 10 Hz. It has a vertical field of view (33 degrees) and 120 meter range
+* 3x RGB global shutter FLIR Backfly cameras modeled by `camera` sensor. Each camera provides 80 degrees of horizontal FOV. Resolution is 808x608 px. According to datasheet, the camera can run on frequencies up to 51 Hz, but the robot doesn't have sufficient computational and communication capacity to transmit such high frequency data. We thus lowered the framerate of the simulated camera to 10 Hz. 
+* XSens MTI-30 IMU: modeled by `imu` sensor.
```

Comment:
This IMU is not currently included in the `model.sdf`.

Reply:
The IMU is now modeled as of https://github.com/osrf/subt/commit/7349e416ef7ccac9a4c3775ecf3057d6e17eef8d